packages/inter-protocol/src/auction/scheduleMath.js

```diff
@@ -24,25 +24,48 @@ export const computeRoundTiming = (params, baseTime) => {
   /** @type {NatValue} */
   const lowestRate = params.getLowestRate();
 
+  const noNextAuction = msg => {
+    console.error(assert.error(msg));
+    return undefined;
+  };
+
   /** @type {RelativeTime} */
   const startDelay = params.getAuctionStartDelay();
-  TimeMath.compareRel(freq, startDelay) > 0 ||
-    Fail`startFrequency must exceed startDelay, ${freq}, ${startDelay}`;
-  TimeMath.compareRel(freq, lockPeriod) > 0 ||
-    Fail`startFrequency must exceed lock period, ${freq}, ${lockPeriod}`;
+  if (TimeMath.compareRel(freq, startDelay) <= 0) {
+    return noNextAuction(
+      X`startFrequency must exceed startDelay, ${freq}, ${startDelay}`,
+    );
+  }
+  if (TimeMath.compareRel(freq, lockPeriod) <= 0) {
+    return noNextAuction(
+      X`startFrequency must exceed lock period, ${freq}, ${lockPeriod}`,
+    );
+  }
+
+  if (startingRate <= lowestRate) {
+    return noNextAuction(
+      X`startingRate ${startingRate} must be more than lowest: ${lowestRate}`,
+    );
+  }
 
-  startingRate > lowestRate ||
-    Fail`startingRate ${startingRate} must be more than lowest: ${lowestRate}`;
   const rateChange = subtract(startingRate, lowestRate);
   const requestedSteps = floorDivide(rateChange, discountStep);
-  requestedSteps > 0n ||
-    Fail`discountStep ${discountStep} too large for requested rates`;
-  TimeMath.compareRel(freq, clockStep) >= 0 ||
-    Fail`clockStep ${TimeMath.relValue(
-      clockStep,
-    )} must be shorter than startFrequency ${TimeMath.relValue(
-      freq,
-    )} to allow at least one step down`;
+
+  if (requestedSteps <= 0n) {
+    return noNextAuction(
+      X`discountStep ${discountStep} too large for requested rates`,
+    );
+  }
+
+  if (TimeMath.compareRel(freq, clockStep) < 0) {
+    return noNextAuction(
+      X`clockStep ${TimeMath.relValue(
+        clockStep,
+      )} must be shorter than startFrequency ${TimeMath.relValue(
+        freq,
+      )} to allow at least one step down`,
+    );
+  }
 
```

Comment:
Consider moving this up to the initial input validation section, since it doesn't depend on any intermediate computations.